scripts/bluetooth

```diff
@@ -94,9 +97,17 @@ for x in range(0, l_r):
         d_name += fs_s[z]
     d_name += " "
 
+pango_formatted_output="<span color=\"{valueColor}\" font_desc=\"{typeface}\">\
```

Comment:
I believe the colors and fonts are in the wrong spot here.  For example when I run the time block: 

```bash
$ /usr/share/i3xrocks/time 
<span color="#E6E1CF"></span><span font_desc="JetBrains Mono Medium 13" color="#E6E1CF"> 12/07 10:16 </span>
```

Notice that the first span is for the color of the icon (the font is defaulted in the i3 config), and the second span specifies the non-icon font and the color of the text.

Reply:
Oh I see… I though that I'd set the same font for both label and value and only override the color of label span,
like (in pseudo code) `<span font=font color=value.color><span color=label.color>label</span> label</span>`
but well, it could break the look if the user's selected font doesn't support labels.